tests/istio_testcases/example_policies/bookinfo-demo/bookinfo-policy-test-methods-2.yaml

```diff
@@ -14,7 +14,7 @@ spec:
             principals: ["cluster.local/ns/default/sa/bookinfo-productpage"]
       to:
         - operation:
-            notMethods: ["GET"]
```

Comment:
keep older version, duplicate with modifications to another test

Reply:
Done